src/Microsoft.Win32.Registry/tests/RegistryKey/RegistryKey_CreateSubKey_str.cs

```diff
@@ -53,6 +41,22 @@ public void NegativeTests()
             });
         }
 
+        [Fact(Skip = "This test fails on Win10 Anniversary Update Issue #10546")]
```

Comment:
Should we just delete the tests rather than disabling them?

If not, please use ActiveIssue instead of Skip.